resolver/resolver.go

```diff
@@ -237,8 +237,8 @@ type ClientConn interface {
 	// UpdateState can be omitted.
 	UpdateState(State) error
 	// ReportError notifies the ClientConn that the Resolver encountered an
-	// error.  The ClientConn will notify the load balancer and begin calling
-	// ResolveNow on the Resolver with exponential backoff.
+	// error. The Resolver itself will handle retrying the resolution, possibly
+	// using an exponential backoff mechanism depending on its implementation.
```

Comment:
I think the comment should say "Resolver should" instead of "Resolver will" because the resolver is to be implemented by users and not grpc itself. The comment should also NOT mention what kind of backoff to use (exponential) unless it's specified in grpc specifications. The comment from the above method can be reused here:

> The resolver should try to resolve the target again. The resolver should use a backoff timer to prevent overloading the server with requests.

Reply:
Done